docs/src/main/sphinx/connector/sql-update-limitation.fragment

```diff
@@ -0,0 +1,32 @@
+### UPDATE
+
+If a ``WHERE`` clause is specified, the ``UPDATE`` operation only works if the
+predicate in the clause can be fully pushed down to the data source.
+
+Only ``UPDATE`` statements with constant assignments and predicates are
+supported. For example, the following statement is supported:
+
+```sql
+UPDATE table SET col1 = 1 WHERE col3 = 1
+```
+
+Arithmetic expressions can not be used with the ``SET`` command. The following
+statement is not supported:
+
+```sql
+UPDATE table SET col1 = col2 + 2 WHERE col3 = 1
+```
+
+``AND`` and ``OR`` operators can not be used in predicates. The following
+statement is not supported:
+
+```sql
+UPDATE table SET col1 = 1 WHERE col3 = 1 AND col2 = 3
+```
+
+All column values of a table row can not be updated simultaneously. The
+following statement is not supported:
+
+```sql
+UPDATE table SET col1 = 1, col2=2, col3=3 WHERE col3 = 1
```

Comment:
for this example to make sense reader needs to know it's a table with just the 3 columns